### ISSUE TITLE
Enhancing Anki: Simultaneous Import for Multiple Decks.

### ISSUE THREAD

**Issue by Say-Hi-To-Me** [ankitects/anki#2821]
- Hello, everyone!
- I have an idea/proposal/plea. 

- Is that possible to add in Anki ability to import notes into few decks simultaneously?

- Because I have some decks with different intervals and settings but the same material. 

- For example, the first deck I use for review only for exposure myself to material which I would need to memorize later on. 
- And in the second one, I use active recall.
- For example, before studying words, it's quite useful to review them without an active recall and get used to them and let the brain get familiar with them.

- I think it might be useful for some users to have such function for studying important to them material.

- And also I don't know, but I suppose it technically is not a hard thing to add to Anki. Because it is the same import but instead of doing it manually twice, it can be done from the first import just by choosing decks into which import must be done.

- If there is a reason why it shouldn't be done, can please somebody explain to me why?

---

TL;DR (by ChatGPT): 
The author suggests adding a feature to Anki that allows importing notes into multiple decks simultaneously. This would be beneficial for users who have different decks with the same material but varying review settings. The author believes it could be a useful and technically feasible addition to Anki and asks for explanations if there are reasons against implementing it.

---

Apologize for the inconvenience due to my English level if there was any.

**Comment by RumovZ** [comment]
Please respect the rules of this project and post questions and suggestions on https://forums.ankiweb.net/.
The manual may also be relevant here: https://docs.ankiweb.net/importing/text-files.html#deck-column
